src/client/Ruby/ms-rest-azure/lib/ms_rest_azure/polling_state.rb

```diff
@@ -98,7 +98,14 @@ def get_operation_error
     
     def get_request(options = {})
       link = @azure_async_operation_header_link || @location_header_link
-      options[:connection] = create_connection(options[:base_uri])
+      faraday_options = {
+          middlewares: request.middlewares,
+          headers: request.headers,
+          log: request.log
```

Comment:
in general, I am curious & I don't know what are the best practices on when to use @ while accessing property and when not? 
Example: request being a property we are accessing it as `request` but in PollingState class azure_async_operation_header_link being property we are accessing it as `@azure_async_operation_header_link`